src/Cloud/weglide/UploadFlight.hpp

```diff
@@ -29,17 +29,17 @@ Copyright_License {
 
 #include <cstdint>
 
-struct WeGlideSettings;
 class Path;
 class CurlGlobal;
 class ProgressListener;
 
 namespace WeGlide {
+struct Pilot;
 
 Co::Task<boost::json::value>
-UploadFlight(CurlGlobal &curl, const WeGlideSettings &settings,
+UploadFlight(CurlGlobal& curl,  const char *url, const struct Pilot &pilot,
```

Comment:
Redundant `struct` keyword, again. And you moved the `&`, breaking the coding style.